<?xml version="1.0" ?>
<launch>
    <!-- See: http://wiki.ros.org/laser_assembler -->

    <group ns="tilting_lidar_scanner">
        <param name="laserScanTopic" type="str" value="/scan" />
        <param name="serialPort" type="str" value="/dev/ttyACM0" />

        <param name="frameID" type="str" value="tilting_lidar_scanner" />

        <param name="xOffset" type="double" value="0" />
        <param name="yOffset" type="double" value="0" />
        <param name="zOffset" type="double" value="1.537" />

        <param name="roll" type="double" value="0" />
        <param name="pitch" type="double" value="0" />
        <param name="yaw" type="double" value="0" />

        <node pkg="pcg_node" type="TiltingLIDARScanner"
            name="TiltingLIDARScanner" output="screen" />
    </group>

    <group ns="hokuyo_node">
        <param name="calibrate_time" type="bool" value="false" />
        <param name="port" type="str" value="/dev/ttyACM1" />
        <param name="min_ang" type="double" value="-2.08621382713" />
        <param name="max_ang" type="double" value="2.09234976768" />
    </group>

    <node pkg="hokuyo_node" type="hokuyo_node"
        name="hokuyo_node" output="screen" />
</launch>
